src/main/resources/swagger/api-docs.yaml

```diff
@@ -3308,6 +3308,34 @@ paths:
         500:
           description: Internal Error
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/importPFB:
+    post:
+      x-passthrough: false
+      tags:
+        - Entities
+      operationId: importPFB
+      summary: Import PFB data from an [Avro](https://avro.apache.org/) file
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/workspaceNamespaceParam'
+        - $ref: '#/parameters/workspaceNameParam'
+        - in: body
+          description: JSON object containing PFB URL
+          name: pfbImportRequest
+          required: true
+          schema:
+            $ref: '#/definitions/PFBRequest'
+      responses:
+        200:
+          description: Successful Request
+        400:
+          description: Bad Request
+        404:
+          description: Source Workspace not found
```

Comment:
nit: it's not really a source workspace (i noticed several of the other import entities also say source workspace. i think it originated from copy entities but for that request i'm wondering what error is returned if the destination workspace is not found). anyway, just fix what you think is relevant without yak shaving